Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerTests.swift

```diff
@@ -338,6 +338,7 @@ class SentryNetworkTrackerTests: XCTestCase {
         XCTAssertEqual(breadcrumb!.data!["response_body_size"] as! Int64, DATA_BYTES_RECEIVED)
         XCTAssertEqual(breadcrumb!.data!["http.query"] as? String, "query=value&query2=value2")
         XCTAssertEqual(breadcrumb!.data!["http.fragment"] as? String, "fragment")
+        XCTAssertNotNil(breadcrumb!.data!["request_start"])
```

Comment:
`m`: Asserting if this is an actual date value would make sense by using the CurrentDateProvider to make the test deterministic.

Reply:
We can assert if this is a NSDate. using CurrentDateProvider instead of `[NSDate date]` is such an overkill to ensure that we're getting the current date.